.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node: ['6', '8', '10', '12', '14', '16']
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm install
```

Comment:
Maybe add [`package-lock.json`](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json) to the repo (it is currently in `.gitignore`) and use [`npm ci`](https://docs.npmjs.com/cli/v7/commands/npm-ci) here instead.

Reply:
Not sure I see a benefit there, I use `npm install` on all my CI setup, specifically not using `npm ci` because it had odd behaviours/bugs related to native modules.